assets/src/blocks/Gallery/GalleryEditor.js

```diff
@@ -147,9 +152,9 @@ const renderView = (attributes, setAttributes) => {
         withoutInteractiveFormatting
         allowedFormats={['core/bold', 'core/italic']}
       />
-      {layout === 'slider' && <GalleryCarousel images={images || []} isEditing />}
-      {layout === 'three-columns' && <GalleryThreeColumns images={images || []} postType={postType} />}
-      {layout === 'grid' && <GalleryGrid images={images || []} />}
+      {layout === 'slider' && <GalleryCarousel isEditing {...galleryProps} />}
```

Comment:
Where does `isEditing` come from?

Reply:
It's just always true in this case, since it's in the editor